polaris-react/src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -26,6 +26,10 @@
   &::after {
     @include track-dashed;
   }
+
+  @media #{$p-breakpoints-sm-down} {
```

Comment:
`.InputWrapper` has a `height: var(--pc-range-slider-thumb-size);` defined on line 18.

Inside the media query we should override it with `height` instead of `min-height`. Unlike the checkbox setting, the content inside the slide won't change so we can override `height` instead of using `min-height`